packages/core/test/unit/query/flux.test.ts

```diff
@@ -156,6 +161,19 @@ describe('Flux Tagged Template', () => {
       // ok expected, undefined is not supported
     }
   })
+  it('fails on empty toString', () => {
+    try {
+      const x = {
+        toString(): string {
+          return ''
+        },
+      }
+      flux`${x}`
+      expect.fail()
+    } catch (_e) {
+      // ok expected, undefined is not supported
+    }
+  })
```

Comment:
why should this case fail? wouldnt it be just like:
```
flux`${''}`
```

Reply:
There is no equivalent of undefined in flux, it is not `""` string. It is IMHO better to fail on undefined.

Reply:
but this case is not about undefined, but empty string?

Reply:
I am sorry, I looked at different lines. Yes, it could return an empty string then, A rare case indeed.